akka-http-core/src/test/scala/akka/http/javadsl/JavaInitializationSpec.scala

```diff
@@ -12,7 +12,7 @@ class JavaInitializationSpec extends WordSpec with Matchers {
 
     "initializes the right field" in {
       akka.http.scaladsl.model.headers.LanguageRange.`*` // first we touch the scala one, it should force init the Java one
-      akka.http.javadsl.model.headers.LanguageRange.ALL // touching this one should not fail
+      val _ = akka.http.javadsl.model.headers.LanguageRange.ALL // touching this one should not fail
```

Comment:
From my understanding it's a regression test for some tricky code which ensures correct initialization of static fields in the Java API.

Blame points to this ticket: https://github.com/akka/akka/issues/20403